### PR TITLE
Adds new tags for lookupValue & lookupValues using token-autocomplete

### PR DESCRIPTION
**Pull request by meg1502**

### CODE REVIEW

File: src/main/java/sirius/biz/codelists/LookupValueSuggestionController.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+import sirius.biz.web.BizController;
+import sirius.kernel.di.std.Part;
+import sirius.kernel.di.std.Register;
+import sirius.web.controller.AutocompleteHelper;
+import sirius.web.controller.Routed;
+import sirius.web.http.WebContext;
+import sirius.web.security.LoginRequired;
+
+/**
+ * Endpoint for autocomplete suggestions for {@link LookupValue} and {@link LookupValues} fields
```
Comment (sabieber) [inline src/main/java/sirius/biz/codelists/LookupValueSuggestionController.java:20]: Optional: Start with a verb as `Provides an endpoint` or `Provides autocomplete suggestions for ...`

File: src/main/java/sirius/biz/codelists/LookupValueSuggestionController.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    /**
+     * Responds with possible suggestions from the given {@link LookupTable} using {@link AutocompleteHelper}.
+     *
+     * @param webContext the web requests calling the autocomplete service
+     * @param tableName  the name of the table for which suggestions should be gathered
+     */
+    @LoginRequired
+    @Routed("/autocomplete/lookuptable/:1")
+    public void suggestFromLookupTable(WebContext webContext, String tableName) {
+        //TODO use new AutocompleteHelper syntax + respect display settings of field
```
Comment (sabieber) [inline src/main/java/sirius/biz/codelists/LookupValueSuggestionController.java:37]: Optional: Please link a Ticket to the TODO

File: src/main/resources/default/taglib/w/lookupValueSingleSelect.html.pasta [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+</div>
+<script>
+    $(document).ready(function () {
+        new TokenAutocomplete({
+            name: '@name',
+            selectMode: SelectModes.SINGLE,
+            selector: '#@id',
+            noMatchesText: '@i18n("template.html.selects.noMatches")',
+            placeholderText: '@i18n("template.html.selects.selection")',
+            suggestionsUri: '/autocomplete/lookuptable/' + '@value.getTableName()'
```
Comment (sabieber) [inline src/main/resources/default/taglib/w/lookupValueSingleSelect.html.pasta:46]:
```suggestion
            suggestionsUri: '/autocomplete/lookuptable/@value.getTableName()'
```

File: src/main/resources/default/taglib/w/lookupValuesMultiSelect.html.pasta [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+    </i:if>
+</div>
+<script>
+    $(document).ready(function () {
+        new TokenAutocomplete({
+            name: '@name',
+            selector: '#@id',
+            noMatchesText: '@i18n("template.html.selects.noMatches")',
+            placeholderText: '@i18n("template.html.selects.selection")',
+            suggestionsUri: '/autocomplete/lookuptable/' + '@values.getTableName()'
```
Comment (sabieber) [inline src/main/resources/default/taglib/w/lookupValuesMultiSelect.html.pasta:49]:
```suggestion
            suggestionsUri: '/autocomplete/lookuptable/@values.getTableName()'
```

**Review by sabieber — APPROVED** [review]

**Review by sabieber — APPROVED** [review]

**Review by jmuscireum — APPROVED** [review]

**Pull request merged**